src/patternfly/components/Masthead/masthead.scss

```diff
@@ -250,6 +249,10 @@ $pf-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
     }
   }
 
+  :where(:not(.pf-c-input-group)) > .pf-c-dropdown {
```

Comment:
do you need the child combinator? also could these styles not just apply to the full height variation, or is there a use case of a regular/non-full-height dropdown that needs the bottom border removed?

```suggestion
  :where(:not(.pf-c-input-group)) .pf-c-dropdown {
```

Reply:
> do you need the child combinator?

Yes, bc `--pf-c-dropdown__toggle--before--BorderBottomColor` belongs to `pf-c-dropdown`, although I couldn't solve needing `>` in the selector. Removing it negates the intended effect. Any ideas?

> also could these styles not just apply to the full height variation, or is there a use case of a regular/non-full-height dropdown that needs the bottom border removed?

@mcoker Here's the original design,  https://marvelapp.com/prototype/5cjajdg/screen/63256958 and there are more iterations showing the context selector styled the same way. As far as I know, this is the intended styling. Also, we can't use `pf-m-plain` as it removes the `active/focus` borders, maybe we need another modifier...

Reply:
> Removing it negates the intended effect

Ah, I see. It works like that, I guess it just introduces some fragility where the dropdown needing to be a direct child of the input group. 

Hmm... just to double check, @mmenestr @mcarrano @mceledonia can you confirm the user dropdown design? It currently looks like this:

<img width="1228" alt="Screen Shot 2022-03-01 at 6 25 12 PM" src="https://user-images.githubusercontent.com/35148959/156271728-2d995abc-90ce-4cea-bd31-a82b2da050eb.png">

I'm wondering if it should look like one of these two:

<img width="1434" alt="Screen Shot 2022-03-01 at 6 28 06 PM" src="https://user-images.githubusercontent.com/35148959/156271725-5df28682-0921-4970-b903-be8b64e6d94c.png">
<img width="1434" alt="Screen Shot 2022-03-01 at 6 27 48 PM" src="https://user-images.githubusercontent.com/35148959/156271727-93013112-f5ce-4d58-a0a3-c8bfa465c550.png">

Which would match the style of other menus like that in the masthead

<img width="1434" alt="Screen Shot 2022-03-01 at 6 28 53 PM" src="https://user-images.githubusercontent.com/35148959/156271719-8353e5cb-0f03-4c63-9e28-6b4aef2f9a16.png">

> maybe we need another modifier

Maybe so... otherwise we might need to add a bunch of exceptions to the styles like we are here, which could potentially get complicated down the road.